README.md

```diff
@@ -1,3 +1,31 @@
-# radiance ☀️
+# radiance #
 
 Highly experimental Solana playing ground.
+
+## Available tooling ##
+
+- [solrays](cmd/solrays), an RPC proxy that exports call latency of a Solana RPC node.
```

Comment:
I prefer to keep as much documentation as possible tied to the code, ie kept in the same repo, so that when the code changes, eg a flag changes, the doc follows in the same commit. I also think this being OSS there are no guarantees for the code or docs to work 😅 so I'm not worried if someone out there picks this up and it breaks.

Reply:
I defer to the license:
>        http://www.apache.org/licenses/LICENSE-2.0
>
>   Unless required by applicable law or agreed to in writing, software
>   distributed under the License is distributed on an "AS IS" BASIS,
>   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.